models/Performance.js

```diff
@@ -174,6 +126,6 @@ module.exports = class Performance {
     }
     const now = moment.utc(new Date().toJSON());
 
-    return moment(performance.openat).isBefore(moment(now)) && moment(now).isBefore(moment(performance.closeat));
+    return moment(new Date(performance.openat)).isBefore(moment(now)) && moment(now).isBefore(moment(new Date(performance.closeat)));
```

Comment:
This references `openat` and `closeat` properties on `performance`. Both the [schema](https://github.com/osumb/challenges/blob/669aca9df48780e92ecbda663df5598529351b06/db/schema.sql#L261-L262) and the `format` method above return camelCase `openAt` and `closeAt`. Where is this coming from?